build.gradle

```diff
@@ -120,6 +120,21 @@ allprojects {
       content { includeGroupByRegex('com\\.splunk\\..*') }
     }
     mavenCentral()
+
+    // ethereum execution spec tests fixtures. Exclusively for ethereum submodule to run ref tests
+    def ethExecSpecTestsRepo = ivy {
```

Comment:
Why is this repository created in a function? The example in https://gist.github.com/ysb33r/9f95bab338b912c45986 just adds it to the repositories.

Reply:
@jframe We are assigning the repository definition to a variable so that it can be used in next section i.e. `exclusiveContent`.